src/core/services/LineFactoryService.ts

```diff
@@ -77,17 +69,14 @@ export class LineFactoryServiceImpl {
     const data = {
       borrower,
       ttl,
-      arbiter: Arbiter_GOERLI,
-      oracle: KibaSero_oracle,
-      factoryAddress: LineFactory_GOERLI,
-      swapTarget: SwapTarget_GOERLI,
+      factoryAddress: LINEFACTORY_GOERLI,
     };
     console.log(data);
     return <TransactionResponse>(
       await this.executeContractMethod(
         data.factoryAddress,
         'deploySecuredLine',
-        [data.oracle, data.arbiter, data.borrower, data.ttl, data.swapTarget],
+        [data.borrower, data.ttl],
```

Comment:
I assume these are being removed because we're going to be plugging in the newly deployed address?